src/main/java/com/google/sps/servlets/utils/Range.java

```diff
@@ -24,15 +24,35 @@ public Range(double inclusiveStart, double exclusiveEnd) {
     this.exclusiveEnd = exclusiveEnd;
   }
 
-  public double getInclusiveStart() {
+  private double getInclusiveStart() {
     return inclusiveStart;
   }
 
-  public double getExclusiveEnd() {
+  private double getExclusiveEnd() {
     return exclusiveEnd;
   }
```

Comment:
Shouldn't these getters be set to public? Otherwise there's no point in having them.

Reply:
+1, the more I think about it we should have these as public